include/rocksdb/secondary_cache.h

```diff
@@ -37,6 +37,9 @@ class SecondaryCacheResultHandle {
 
   // Return the size of value
   virtual size_t Size() = 0;
+
+  // Returns whether the handle is erased in the secondary cache.
+  virtual bool IsErasedFromSecondaryCache() = 0;
```

Comment:
I feel that the property doesn't quite fit the cache returned handle. Handle is mainly used for dealing the data itself, rather than communicating status related to Lookup() itself. I think it might be better to add output argument in Lookup() instead.

Reply:
Let me implement this approach.

Reply:
https://github.com/gitbw95/rocksdb/blob/main/cache/lru_cache.cc#L756 shows that Promote is called in WaitAll. I don't have a good way to pass the output argument value to this Promote. [Let's discuss with @anand1976 about this ].